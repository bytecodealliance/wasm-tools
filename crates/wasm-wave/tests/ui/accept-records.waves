basic-record({ required: 1 });
basic-record({ required: 1, optional: none });
basic-record({ required: 1, optional: none, });
basic-record({ optional: none, required: 1 });
basic-record({ required: 1, optional: some(2) });
basic-record({ optional: some(2), required: 1, });
optional-fields-record({:});
optional-fields-record({ : });
optional-fields-record({a:none,b:none});
optional-fields-record({b:none,});
optional-fields-record({a:some(1)});
optional-fields-record({b:some(2)});
optional-fields-record({b:some(2),a:some(1)});
keyword-fields-record({true: true, false: false, some: some(1), none: none, ok: ok, err: err, inf: inf, nan: nan});