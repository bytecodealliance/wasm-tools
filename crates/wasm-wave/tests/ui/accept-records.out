basic-record(rec: {required: 1})
basic-record(rec: {required: 1})
basic-record(rec: {required: 1})
basic-record(rec: {required: 1})
basic-record(rec: {required: 1, optional: some(2)})
basic-record(rec: {required: 1, optional: some(2)})
optional-fields-record(rec: {:})
optional-fields-record(rec: {:})
optional-fields-record(rec: {:})
optional-fields-record(rec: {:})
optional-fields-record(rec: {a: some(1)})
optional-fields-record(rec: {b: some(2)})
optional-fields-record(rec: {a: some(1), b: some(2)})
keyword-fields-record(rec: {true: true, false: false, some: some(1), ok: ok, err: err, inf: inf, nan: nan})
